utils/led.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * led.h
+ *
+ *  Created on: 29-Jan-2021
+ *      Author: hp
+ */
+
+#ifndef UTILS_LED_H_
+#define UTILS_LED_H_
+
+void led_blink(led_error_type_t blinks);
+void led_set_high(void);
```

Comment:
I'd rename this to `led_turn_on()` or `led_on()`.

Reply:
Sure! I changed it to `led_turn_on()`

Reply:
Great - thanks!